aiohttp/http_writer.py

```diff
@@ -24,6 +25,16 @@
 __all__ = ("StreamWriter", "HttpVersion", "HttpVersion10", "HttpVersion11")
 
 
+MIN_PAYLOAD_FOR_WRITELINES = 2048
+UNSAFE_WRITELINES = (3, 13, 0) <= sys.version_info < (3, 13, 2) or sys.version_info < (
```

Comment:
The part after `or` will always evaluate to `False`. Because, for 3.12.0 the first part will be `True`. The check must be more granular if we want this to work.

Reply:
Oh... nevermind. I missed that it's a double-sided check with `<= x <`.